examples/check_integration_error.py

```diff
@@ -41,90 +45,87 @@
 clean_data["t2":0] *= 0.5
 fake_data_noise_std = 2.0
 clean_data.reorder(["ph1", "ph2", "vd"])
-bounds = (0, 200)  # seem reasonable to me
 result = 0
 n_repeats = 100
 all_results = ndshape(clean_data) + (n_repeats, "repeats")
 all_results.pop("t2").pop("ph1").pop("ph2")
 all_results = all_results.alloc()
 all_results.setaxis("vd", clean_data.getaxis("vd"))
 print("shape of all results", ndshape(all_results))
-for j in range(n_repeats):
-    data = clean_data.C
-    data.add_noise(fake_data_noise_std)
-    # at this point, the fake data has been generated
-    data.ft(["ph1", "ph2"])
-    # {{{ usually, we don't use a unitary FT -- this makes it unitary
-    data /= 0.5 * 0.25  # the dt in the integral for both dims
-    data /= sqrt(ndshape(data)["ph1"] * ndshape(data)["ph2"])  # normalization
-    # }}}
-    dt = diff(data.getaxis("t2")[r_[0, 1]]).item()
-    data.ft("t2", shift=True)
-    # {{{
-    data /= sqrt(ndshape(data)["t2"]) * dt
-    error_pathway = (
-        set((
-            (j, k)
-            for j in range(ndshape(data)["ph1"])
-            for k in range(ndshape(data)["ph2"])
-        ))
-        - set(excluded_pathways)
-        - set([(signal_pathway["ph1"], signal_pathway["ph2"])])
+with figlist_var() as fl:
+    for j in range(n_repeats):
+        data = clean_data.C
+        data.add_noise(fake_data_noise_std)
+        # at this point, the fake data has been generated
+        data.ft(["ph1", "ph2"], unitary=True)
+        dt = diff(data.getaxis("t2")[r_[0, 1]]).item()
+        data.ft("t2", shift=True)
+        data /= sqrt(ndshape(data)["t2"]) * dt
+        s_int, frq_slice = frequency_domain_integral(
+            data,
+            signal_pathway=signal_pathway,
+            excluded_pathways=excluded_pathways,
+            indirect="vd",
+            fl=fl,
+            return_frq_slice=True,
+        )
+        manual_integration = select_pathway(data, signal_pathway)[
```

Comment:
here I am integrating over the same range as the function but manually calculating the error for comparison of the error as the name implies

Reply:
But the point of the function, and what I think the legend indicates, is to compare the automatically determined bounds versus bounds that you have yourself manually chosen

Reply:
But ine of the things that we are testing in the example, and what I think the legend indicates, is to compare the automatically determined bounds versus bounds that you have yourself manually chosen

Reply:
But one of the things that we are testing in the example, and what I think the legend indicates, is to compare the automatically determined bounds versus bounds that you have yourself manually chosen